doc/source/index.rst

```diff
@@ -103,7 +103,7 @@ immediately and Pythonically.
 Calling MAPDL Pythonically
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 MAPDL functions can be called directly from a :class:`Mapdl
-<ansys.mapdl.core.mapdl._MapdlCore>` instance in a Pythonic manner. This is to
+<ansys.mapdl.core.mapdl.MapdlBase>` instance in a Pythonic manner. This is to
```

Comment:
reading the description I did not get the impression that this was a user-facing change, only an internal refactoring. So I am surprised to see doc changes

Reply:
That's a fair point... Thing is the previous known as `_MapdlCore` has been renamed. We have the issue that the general class (`_MapdlCore/MapdlBase` cannot be used directly) but still we need to refer to it somehow. I guess the classes structure is not the best....